model/ConceptProperty.php

```diff
@@ -22,10 +24,11 @@ class ConceptProperty
      * @param string $prop property type eg. 'rdf:type'.
      * @param string $label
      */
-    public function __construct($prop, $label, $super=null, $sort_by_notation=false)
+    public function __construct($prop, $label, $tooltip=null, $super=null, $sort_by_notation=false)
```

Comment:
One could document any new parameter(s) here.
On a side note, it's also a bit risky to add a parameter in the middle of a constructor call parameter list, but in this case it does not cause trouble. It may very well be intended, though, if one thinks it is a more important one to have.

Reply:
You're right, the phpdoc declaration for the new parameter is missing.

I added the parameter in the middle because in PHP, only the last parameters can be omitted. Since there is a need to call the constructor both with and without the last two parameters, this was the only clean option. I did check all calls to the constructor.